doc/rados/operations/erasure-code.rst

```diff
@@ -47,13 +47,13 @@ requires at least three hosts:
 
    ABCDEFGHI
 
-Erasure code profiles
+Erasure-code profiles
 ---------------------
 
-The default erasure code profile can sustain the loss of two OSDs. This erasure
-code profile is equivalent to a replicated pool of size three, but requires
-2TB to store 1TB of data instead of 3TB to store 1TB of data. The default
-profile can be displayed with this command:
+The default erasure-code profile can sustain the loss of two OSDs. This erasure-
```

Comment:
the overlapping loss of two OSDs without losing data.

Reply:
Good note.